app/src/android/input.h

```diff
@@ -91,6 +91,8 @@ enum android_input_event_type {
 
 /**
  * Key event actions.
+ *
+ * Also keep `android_keyevent_action_descriptions` updated.
```

Comment:
This file is copied from AOSP: [`include/android/input.h`](https://android.googlesource.com/platform/frameworks/native/+/refs/heads/master/include/android/input.h). Please do not modify it.